auction-server/api-types/src/opportunity.rs

```diff
@@ -343,59 +343,59 @@ pub enum OpportunityParamsV1ProgramSvm {
 #[derive(Serialize, Deserialize, ToSchema, Clone, PartialEq, Debug, ToResponse)]
 #[serde(rename_all = "snake_case")]
 pub enum FeeToken {
-    InputToken,
-    OutputToken,
+    SearcherToken,
+    UserToken,
 }
 
 #[serde_as]
 #[derive(Serialize, Deserialize, ToSchema, Clone, PartialEq, Debug, ToResponse)]
 #[serde(tag = "side_specified")]
 pub enum QuoteTokens {
-    #[serde(rename = "input")]
-    #[schema(title = "input_specified")]
-    InputTokenSpecified {
-        /// The token that the user wants to receive
+    #[serde(rename = "searcher")]
+    #[schema(title = "searcher_specified")]
```

Comment:
```suggestion
    #[schema(title = "searcher")]
```

Reply:
<img width="911" alt="Screenshot 2025-01-29 at 2 57 59 PM" src="https://github.com/user-attachments/assets/d6de3239-7649-4eb0-8cd5-63bc14da04ca" />

this will just change this. i think it's clearer in the api docs if this reads as "searcher_specified" instead of just "searcher"

Reply:
okok